### ISSUE TITLE
Settings: AMP toggle missing for simple and AT sites

### ISSUE THREAD

**Issue by samiff** [Automattic/wp-calypso#38842]
#### Steps to reproduce
* Visit `Manage > Settings > Performance (tab)`: https://wordpress.com/settings/performance/__SITE_ADDRESS__

#### What I expected
* On simple sites, a toggle to show that allows turning off AMP per the support guide: https://en.support.wordpress.com/amp-accelerated-mobile-pages/
* On AT sites, a method or information for deactivating AMP for the site.

#### What happened instead
* For simple sites there should be a toggle to turn off AMP.
* For AT sites, I'm not sure of the expected behavior. If deactivating the AMP plugin is the only method, then we need to update the support guide with that information.
* #36371 notes that the AMP toggle is not found on AT sites. I wasn't able to find an issue for the toggle not showing as expected for simple sites.
* #36319 notes that simple sites don't see the AMP design options in Customizer which I'm still able to reproduce. 

#### Screenshot / Video
Expected:

![Screen Shot 2020-01-14 at 3 14 23 PM](https://user-images.githubusercontent.com/15803018/72387354-92b2ff80-36e0-11ea-8912-67c51514e6a8.png)

Observed:

![Screen Shot 2020-01-14 at 3 14 51 PM](https://user-images.githubusercontent.com/15803018/72387374-a199b200-36e0-11ea-8555-0b3ff7da6444.png)

#### Context / Source
#user-report 17913208-hc

**Comment by rachelmcr** [comment]
While testing an issue with AMP today, I came across this issue. I couldn't reproduce the issue with simple sites reported here. I confirmed that once I launched my new site and it was public (per the [support doc](https://en.support.wordpress.com/amp-accelerated-mobile-pages/)) AMP was enabled and the AMP setting was visible in my site's performance settings.

I noticed that if I changed my site's visibility to hidden/private that setting was removed; is it possible the simple site where this was observed was not public?
